### PR TITLE
Adds and Deletes posting successfully to batch edit process

### PR DESCRIPTION
**Pull request by adamjarling**
This finalizes the process of preparing and submitting Batch Edit add and delete items to the GraphQL API.  

I tried to simplify some of contents of `assets/js/components/BatchEdit/About/Confirmation.jsx`, and how we're prepping data in `assets/js/components/BatchEdit/About/About.jsx` as well. 

image

### CODE REVIEW

**Comment by kdid** [comment]
@adamjarling - In the add and remove screens in your screenshots above, I'd show users the Role Label rather than the Role Id. (There won't be any ambiguity regarding roles so they won't need to see the id to tell them apart).

**Comment by adamjarling** [comment]
@kdid @davidschober @laura-alagna Any value in displaying the Role Scheme in the above screenshot?

**Comment by davidschober** [comment]
I don’t think so.

**Comment by kdid** [comment]
Let's see if we can get it working if we add a `query` to the query. Ex: 

` "{\"query\":{\"term\":{\"workType.id\": \"IMAGE\"}}}"`

**Comment by adamjarling** [comment]
Yea, it needs to be set up like it currently is for the front-end to work, but I'll wrap the final POST to the mutation in a `query: { ...}`.

File: assets/js/components/BatchEdit/About/Confirmation.jsx [hunk 1]
```diff
@@ -107,105 +90,61 @@ const BatchEditConfirmation = ({
...
-                      </li>
-                    )
-                )}
-            </ul>
-          </section> */}
+          {hasAdds && (
+            <section className="content">
+              <div css={headerWrapper}>
+                <FontAwesomeIcon icon="plus" size="2x" />
+                <span class="subtitle">Adding</span>
```
Comment (katdivyareddy10) [inline assets/js/components/BatchEdit/About/Confirmation.jsx:97]: can you change this to className?

File: assets/js/components/BatchEdit/About/Confirmation.jsx [hunk 2]
```diff
@@ -107,105 +90,61 @@ const BatchEditConfirmation = ({
...
+                );
+              })}
+            </section>
+          )}
+
+          {hasDeletes && (
+            <section className={`content ${hasAdds ? "py-6" : ""}`}>
+              <div css={headerWrapper}>
+                <FontAwesomeIcon icon="minus-square" size="2x" />
+                <span class="subtitle">Removing</span>
```
Comment (katdivyareddy10) [inline assets/js/components/BatchEdit/About/Confirmation.jsx:121]: Here as well

File: assets/js/components/BatchEdit/About/Confirmation.jsx (hunk 1 above)
Comment (adamjarling) [reply assets/js/components/BatchEdit/About/Confirmation.jsx]: Will do

**Comment by kdid** [comment]
Functionality works now from the UI. Looks good.

**Pull request merged**